src/modules/main/modals/CreateColumn.vue

```diff
@@ -203,8 +201,8 @@ export default {
 				{ id: 'number-stars', label: t('tables', 'Stars rating') },
 				{ id: 'number-progress', label: t('tables', 'Progress bar') },
 
-				// { id: 'selection', label: t('tables', 'Selection') },
-				// { id: 'selection-multi', label: t('tables', 'Multiselect') },
+				{ id: 'selection', label: t('tables', 'Selection') },
+				{ id: 'selection-multi', label: t('tables', 'Multiselect') },
```

Comment:
Calling @jancborchardt for the wording here. My suggestion would be "Single select" and "Multiple select" to make it more clear what the difference is. This is also how airtable for example names them https://support.airtable.com/docs/supported-field-types-in-airtable-overview

Reply:
Yeah good point – let’s do it like that, more clear and then also people coming from Airtable will understand it easily. :)